README.md

```diff
@@ -492,7 +492,7 @@ ort {
     storages {
       postgresStorage {
         url = "jdbc:postgresql://example.com:5444/database"
-        schema = "schema"
+        schema = "public"
```

Comment:
Commit message:

```
postgres: Use default schema if none defined

The schema now is an optional parameter as the postgres default of "public"
is used.
```

There is no need to repeat too much of the README docs here.